src/pymor/operators/constructions.py

```diff
@@ -556,12 +556,12 @@ def apply(self, U, mu=None):
         if not self.transposed:
             return self._array.lincomb(U.data)
         else:
-            return self.range.make_array(U.dot(self._array))
+            return self.range.make_array(U.conj().dot(self._array))
 
     def apply_transpose(self, V, mu=None):
         assert V in self.range
         if not self.transposed:
-            return self.source.make_array(self._array.dot(V).T)
+            return self.source.make_array(self._array.conj().dot(V).T)
```

Comment:
This change would be needed, since in case `not self.transposed` we want to have a linear operator, and `apply_transpose` is not supposed to do complex conjugation?